drivers/hmis/app/graphql/types/hmis_schema/project.rb

```diff
@@ -26,6 +26,12 @@ def self.configuration
       Hmis::Hud::Project.hmis_configuration(version: '2024')
     end
 
+    # check for the most minimal permission needed to resolve this object
+    def self.authorized?(object, ctx)
+      permission = :can_view_project
+      super && GraphqlPermissionChecker.current_permission_for_context?(ctx, permission: permission, entity: object)
+    end
```

Comment:
This might break some things, we should do a close QA on it. It may mean we need to change some places that resolve Project to resolve PublicProjectInfo .

Reply:
We could skip this if you think it's too risky; I figured it was worth adding a check since Project has so many edges.

Reply:
no this is good you can ignore my comment. I looked and it is not risky